packages/api/src/controllers/reviews/v1/mod.rs

```diff
@@ -88,21 +93,46 @@ impl ReviewControllerV1 {
     pub async fn update(&self, id: i64, params: ReviewUpdateRequest) -> Result<Json<Review>> {
         tracing::debug!("update review: {:?} {:?}", id, params);
 
-        Ok(Json(Review::default()))
+        let fetched_review = self.review_repo
+            .update(
+                id,
+                ReviewRepositoryUpdateRequest {
+                    name: Some(params.name),
+                    image: Some(params.image),
+                    review: Some(params.review),
+                },
+            )
+            .await?;
+
+        Ok(Json(fetched_review))
     }
 
     pub async fn find(
         &self,
         ReviewQuery { size, bookmark, .. }: ReviewQuery,
     ) -> Result<Json<ReviewGetResponse>> {
-        let _size = size;
+        let _size = size as i64;
         let _bookmark = bookmark;
 
+        let mut total_count: i64 = 0;
+        let query = ReviewSummary::base_sql_with("limit $2 offset $3");
+
         tracing::debug!("find query");
 
+        let items: Vec<ReviewSummary> = sqlx::query(&query)
+            .bind(_size)
+            .bind(_size * (_bookmark.unwrap_or("1".to_string()).parse::<i64>().unwrap() - 1))
+            .map(|r: PgRow| {
+                use sqlx::Row;
+                total_count = r.get("total_count");
+                r.into()
+            })
+            .fetch_all(&self.pool)
+            .await?;
```

Comment:
```suggestion
        let items: Vec<ReviewSummary> = ReviewSummary::query_builder().limit(param.size()).page(param.page()).query()
            .map(|r: PgRow| {
                use sqlx::Row;
                total_count = r.get("total_count");
                r.into()
            })
            .fetch_all(&self.pool)
            .await?;
```

you can use `param: ReviewQuery` instead of destructing.
And the ReviewQuery provides two functions to convinient conversion to sqlx params; `size(&self)` and `page(&self)`